Resources/views/Collector/propel.html.twig

```diff
@@ -73,33 +73,33 @@
             <tr><td>No queries.</td></tr>
         {% else %}
             {% for i, query in collector.queries %}
-            <tr>
-                <td>
-                    <a name="propel-query-{{ i }}" ></a>
-                    <code>{{ query.sql|format_sql }}</code>
-                    {% if app.request.query.has('query') and app.request.query.get('query') == i %}
-                        <div class="SQLExplain">
-                        {% render controller('PropelBundle:Panel:explain', {
-                            'token': token,
-                            'panel': 'propel',
-                            'query': app.request.query.get('query'),
-                            'connection': app.request.query.get('connection')
-                        }) %}
-                        </div>
-                    {% endif %}
-                    <div class="SQLInfo">
-                        Time: {{ query.time }} - Memory: {{ query.memory }} - Connection: {{ query.connection }}
-
-                        {% if app.request.query.get('query', -1) != i %}
-                            - <a href="{{ path('_profiler', {'panel': 'propel', 'token': token, 'connection': query.connection, 'query': i}) }}#propel-query-{{ i }}">Explain the query</a>
+                <tr>
+                    <td>
+                        <a name="propel-query-{{ i }}" ></a>
+                        <code>{{ query.sql|format_sql }}</code>
+                        {% if app.request.query.has('query') and app.request.query.get('query') == i %}
+                            <div class="SQLExplain">
+                                {{ render(controller('@Propel/Panel/explain', {
```

Comment:
Isn't it inconsistent with line 104? Here we replace the syntax to use slashes, but for the same render(controller(...)) in line 104 we keep the syntax with the colons, is there a reason?

Reply:
Alex had the same question. But I saw that changing 104 line by using @ caused the error in the logs. It was saying that the controller is not found. I can try to change the 82 line back to use :.

Reply:
I changed the 82 line to PropelBundle:Panel:explain and the Symfony profiler seems to be working fine.